src/amo/api/index.js

```diff
@@ -115,17 +120,19 @@ export function getReviews(
 type GetLatestReviewParams = {|
   addon: number,
   user: number,
+  version: number,
 |};
 
 export function getLatestUserReview(
-  { user, addon }: GetLatestReviewParams = {}
+  { user, addon, version }: GetLatestReviewParams = {}
 ) {
   return new Promise((resolve) => {
-    if (!user || !addon) {
-      throw new Error('Both user and addon must be specified');
+    if (!user || !addon || !version) {
+      throw new Error('user, addon, and version must be specified');
     }
-    // The API will only return the latest user review for this add-on.
-    resolve(getReviews({ user, addon }));
+    // The API will only return the latest user review for this add-on
+    // and version.
+    resolve(getReviews({ user, addon, version }));
```

Comment:
Sort of irking me these aren't alphabetised everywhere but I mean it's not that big of a deal.